examples/SmartShop/contracts/BuyerContract.aes

```diff
@@ -9,28 +9,20 @@ contract TransportInterface =
   entrypoint check_courier_location : () => string
   
 payable contract Buyer =
```

Comment:
Why did you remove the `state` now you need to pass seller contract and the transport contract in every function?

Reply:
Fixed in the new commit